testlist/mips-riscv-x86-xtensa.dat

```diff
@@ -5,3 +5,5 @@
 -nr5m100-nexys4:nsh
 
 /x86
+
+/xtensa
```

Comment:
Should we move the change to:
https://github.com/apache/incubator-nuttx-testing/blob/master/testlist/full.dat
since xtensa-esp32-gcc-toolchain is just used for the nightly build.

Reply:
You are right. Firstly enable xtensa esp32 boards build in full testlit done.